src/lib/IlmThread/IlmThreadExport.h

```diff
@@ -3,15 +3,27 @@
 // Copyright (c) Contributors to the OpenEXR Project.
 //
 
+// See docs/SymbolVisibility.md for more discussion
+
+/// \addtogroup ExportMacros
+/// @{
 #if defined(OPENEXR_DLL)
-    #if defined(ILMTHREAD_EXPORTS)
-	    #define ILMTHREAD_EXPORT __declspec(dllexport)
-        #define ILMTHREAD_EXPORT_CONST extern __declspec(dllexport)
-    #else
-	    #define ILMTHREAD_EXPORT __declspec(dllimport)
-	    #define ILMTHREAD_EXPORT_CONST extern __declspec(dllimport)
-    #endif
+#    if defined(ILMTHREAD_EXPORTS)
+#	    define ILMTHREAD_EXPORT __declspec(dllexport)
+#    else
+#	    define ILMTHREAD_EXPORT __declspec(dllimport)
+#    endif
+#    define ILMTHREAD_EXPORT_TYPE
+#    define ILMTHREAD_HIDDEN
 #else
-    #define ILMTHREAD_EXPORT
-    #define ILMTHREAD_EXPORT_CONST extern const
+#    ifndef _MSC_VER
+#        define ILMTHREAD_EXPORT __attribute__ ((__visibility__ ("default")))
+#        define ILMTHREAD_EXPORT_TYPE __attribute__ ((__visibility__ ("default")))
+#        define ILMTHREAD_HIDDEN __attribute__ ((__visibility__ ("hidden")))
+#    else
+#	     define ILMTHREAD_EXPORT
```

Comment:
indentation?

Reply:
yeah, the old files somehow had tabs in the spaces (my editor is spaces only)